dropwizard-json-logging/src/main/java/io/dropwizard/logging/json/layout/AccessJsonLayout.java

```diff
@@ -67,13 +71,20 @@ public AccessJsonLayout(JsonFormatter jsonFormatter, TimestampFormatter timestam
             .addNumber("status", isIncluded(AccessAttribute.STATUS_CODE), event::getStatusCode)
             .add("userAgent", isIncluded(AccessAttribute.USER_AGENT), () -> event.getRequestHeader(USER_AGENT))
             .add("version", jsonProtocolVersion != null, jsonProtocolVersion)
+            .addMap("requestAttributes", !requestAttributes.isEmpty(),
+                () -> includeRequestAttributes(requestAttributes, event))
             .build();
     }
 
     private boolean isIncluded(AccessAttribute attribute) {
         return includes.contains(attribute);
     }
 
+    private Map<String, String> includeRequestAttributes(Set<String> requestAttributeNames, IAccessEvent event) {
```

Comment:
For consistency should be renamed to `filterRequestAttributes` to align with `filterMDC` and `filterHeaders`